client/src/components/Price.jsx

```diff
@@ -1,27 +1,30 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Button, UncontrolledTooltip } from 'reactstrap';
 import TypeModal from './TypeModal';
-import Star from './Star';
 
-const Price = ({ types, price, addNewType }) => {
-  return (
-    <div className="price">
-      <div id="price">
-        {price}
-      </div>
-      <div>
-      •
-      </div>
-      <div id="types">
-      { types ? types.map(i => <a href ="#">{i}</a>) : '' } {/* eslint-disable-line */}
-      </div>
-      <div>
-        <TypeModal addNewType={addNewType} className="typeModal" />
-      </div>
+const uniqid = require('uniqid');
```

Comment:
What would a best practice be when assigning id's to a component to avoid the unique key error? I found this 'uniqid' library on npm after some googling. Would you recommend this method?